code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -336,3 +336,88 @@
+
+/obj/item/clothing/shoes/cowboy
+	name = "cowboy boots"
+	desc = "A small sticker lets you know they've been inspected for snakes, It is unclear how long ago the inspection took place..."
+	icon_state = "cowboy_brown"
+	permeability_coefficient = 0.05 //these are quite tall
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/shoes
+	custom_price = 35 //poor assistants cant afford 50 credits
+	var/list/occupants = list()
+	var/max_occupants = 4
+
+/obj/item/clothing/shoes/cowboy/Initialize()
+	. = ..()
+	if(prob(2))
+		var/mob/living/simple_animal/hostile/retaliate/poison/snake/bootsnake = new/mob/living/simple_animal/hostile/retaliate/poison/snake(src)
+		occupants += bootsnake
+
+
+/obj/item/clothing/shoes/cowboy/equipped(mob/living/carbon/user, slot)
+	. = ..()
+	if(slot == SLOT_SHOES)
+		if(occupants)
+			for(var/mob/living/occupant in occupants)
+				occupant.forceMove(get_turf(user))
+				to_chat(user, "<span class='userdanger'> You feel a sudden stabbing pain in your [pick("foot", "toe", "ankle")]!</span>",
+				"<span class='warning'>")
+				user.visible_message("[user] recoils as something slithers out of [src].</span>")
+				user.Knockdown(20) //Is one second paralyze better here? I feel you would fall on your ass in some fashion.
+				user.apply_damage(5, BRUTE, pick(BODY_ZONE_R_LEG, BODY_ZONE_L_LEG))
+				if(istype(occupant, /mob/living/simple_animal/hostile/retaliate/poison))
+					user.reagents.add_reagent(/datum/reagent/toxin, 4)
+				occupants -= occupant
+
+/obj/item/clothing/shoes/cowboy/MouseDrop_T(mob/living/target, mob/user)
+	. = ..()
+	if(target.stat == DEAD)
+		return ..()
+	if(occupants.len >= max_occupants)
+		to_chat(user, "<span class='notice'>[src] seems to be full!</span>")
+		return ..()
+	if(istype(target, /mob/living/simple_animal/hostile/retaliate/poison/snake) || istype(target, /mob/living/simple_animal/hostile/headcrab) || istype(target, /mob/living/carbon/alien/larva))
+		occupants += target
+		target.forceMove(src)
+		to_chat(user, "<span class='notice'>[target] slithers into [src]</span>")
+
+/obj/item/clothing/shoes/cowboy/container_resist(mob/living/user)
+	if(!do_after(user, 10, target = user))
+		return
+	user.forceMove(get_turf(src))
+	occupants -= user
+
+/obj/item/clothing/shoes/cowboy/white
+	name = "white cowboy boots"
+	icon_state = "cowboy_white"
+
+/obj/item/clothing/shoes/cowboy/black
+	name = "black cowboy boots"
+	desc = "You get the feeling someone might have been hanged in these boots."
+	icon_state = "cowboy_black"
+
+/obj/item/clothing/shoes/cowboy/fancy
+	name = "bilton wrangler boots"
+	desc = "A pair of authentic haute couture boots from Japanifornia. You doubt they have ever been close to cattle."
+	icon_state = "cowboy_fancy"
+	permeability_coefficient = 0.08
+
+/obj/item/clothing/shoes/cowboy/lizard
+	name = "lizard skin boots"
+	desc = "You can hear a faint hissing from inside the boots, you hope it is just a mournful ghost."
```

Comment:
comma splice

Reply:
I have changed out the comma for a semicolon. Do  you consider this acceptable? Perhaps you have a suggestion for a conjunction you'd like to see instead?

Thank you for reviewing my PR!

Reply:
That's fine.